sys/saul_reg/saul_reg.c

```diff
@@ -126,3 +128,12 @@ int saul_reg_write(saul_reg_t *dev, phydat_t *data)
     }
     return dev->driver->write(dev->dev, data);
 }
+
+int saul_reg_test(saul_reg_t *dev)
+{
+
+    if (dev == NULL) {
+        return -ENODEV;
+    }
+    return dev->driver->test(dev->dev);
```

Comment:
I just did it the same way than the other calls. I can change if you prefer a NULL pointer instead of the stub.